trade-finance-manager-ui/server/nunjucks-configuration/filter-userIsInTeam.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Nunjucks filter to tell whether or not a user
+ * is in a specific team by the team id
+ * @param {object} user - The user object
+ * @param {import('../types/teamIds').TeamId} teamId - The teamId
+ * @returns {boolean}
+ */
+const userIsInTeam = (user, teamId) => user.teams !== undefined && user.teams.some(({ id }) => teamId === id);
```

Comment:
Originally, I introduced it instead of using the existing functions because it looked like `teams` sometimes didn't exist on `user`. However, I found that this was because of the component tests being written incorrectly, so I updated the component tests which then allowed me to use the existing function